cli/azd/internal/tracing/events/events.go

```diff
@@ -15,18 +15,5 @@ const CommandEventPrefix = "cmd."
 // Prefix for vsrpc events.
 const VsRpcEventPrefix = "vsrpc."
 
-// BicepInstallEvent is the name of the event which tracks the overall bicep install operation.
-const BicepInstallEvent = "tools.bicep.install"
-
-// GitHubCliInstallEvent is the name of the event which tracks the overall GitHub cli install operation.
-const GitHubCliInstallEvent = "tools.gh.install"
-
-// PackCliInstallEvent is the name of the event which tracks the overall pack cli install operation.
-const PackCliInstallEvent = "tools.pack.install"
```

Comment:
is this for azd package?

Reply:
I think this was more about the `pack` CLI tool from the buildpacks project (as an alternative to oryx) for producing docker images from app code without a dockerfile.